api/bin/chainflip-lp-api/src/main.rs

```diff
@@ -140,6 +141,7 @@ pub trait Rpc {
 		id: OrderIdJson,
 		tick: Option<Tick>,
 		amount_change: IncreaseOrDecrease<NumberOrHex>,
+		details: Option<OrderScheduleDetails<BlockNumber>>,
```

Comment:
Shouldn't this be called something like scheduling, or schedule_details..., details is abit unclear, given these are visible to external users via the jsonrpcs.

Reply:
Good point 👍